admin/Dockerfile.qa

```diff
@@ -30,6 +27,6 @@ USER docker:docker
 # Unicode is necessary for some tools like "black" to work.
 ENV LC_ALL=C.UTF-8 LANG=C.UTF-8
 
-CMD ( echo docker | sudo -S -p "Running pip with sudo" $PYTHON -m pip install -e . ) && \
-  ( echo docker | sudo -S -p "Running chmod with sudo" chown -R docker:docker /gcovr ) && \
-  make qa
+CMD ( echo docker | sudo -S -p "Running chmod with sudo..." chown -R docker:docker /gcovr/gcovr ) && \
+  echo "\ndone\nStarting test..." && \
+  python3 -m nox --envdir /tmp/envs --non-interactive
```

Comment:
This will prevent caching of the Nox environments – each time the container runs, it will get a new tmp, and force all dependencies to be downloaded anew. This is going to be especially noticeable for the sizeable packages like cmake which weighs about 20MB.

But I understand that there might be clashes if the envs have the same names as the hosts's envs. Could we avoid this by using `--envdir ./.nox-containerized` instead?

Reply:
I was alos thinking about this. We can't use the default directory and we need to use a different directory for each docker cointainer. I'll change this.